virt/virt-4-8-release-notes.adoc

```diff
@@ -78,6 +78,8 @@ The SVVP Certification applies to:
 
 //CNV-12273 CDI now follows the OpenShift cluster-wide proxy configuration when importing virtual machine disk images.
 
+* You can now prepare a custom disk image, based on an existing disk image, for use as a boot source. Using the {VirtProductName} console, you can customize any template that has a defined source by running a wizard. The wizard steps you through the process of preparing a custom disk image, creating a boot source from the custom disk image, and attaching the boot source to a custom template.
```

Comment:
I might rephrase the second sentence to: `You can customize any template that has a defined source by using a wizard in the {VirtProductName} console.`

Reply:
Done.